### DISCUSSION TITLE
How do I open a previous session?

### DISCUSSION THREAD

**Discussion by n3ry7** [jackMort/ChatGPT.nvim#90] [Q&A]
How to select a previous session in sessions window?

**Comment by pitkling** [comment]
When you have selected the session, simply press `<Space>`. Also, I think yesterday a pull request was merged that allows to change the key from `<Space>` to something else, see Readme.md.

**Reply by Dolvur** [reply]
Do you know why "Enter" is used for changing the settings but "Space" is used for selecting session? Shouldn't these use the same key since they are both additional options that appear when using `<C-o>`?
